src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -823,6 +823,8 @@ private bool ShouldTryToReconnect(Exception ex)
 
                     return true;
 
+                case DatabaseDisabledException:
+                case AllTopologyNodesDownException:
```

Comment:
I think its better  go to default case in such exception so the subscription will be terminated when `MaxErroneousPeriod` is hit

Reply:
Done